test-suite-ports.md

```diff
@@ -16,4 +16,4 @@ For this reason, we maintain an overview of the ports used by the integration te
 | metrics-util         | N.A. (no integration tests, only unit tests) |
 | serverless           | N.A. (no integration tests, only unit tests) |
 | shared-metrics       | app under test: 7215, mock agent: 7211 |
-
+| opentelemetry-exporter | app under test: 3215, mock back end: 10443 |
```

Comment:
Port 3215 is already in use by `collector`, can we use a different port?

Reply:
Good catch. 
I also added the collector to the list with the port 3215.